pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -417,7 +417,7 @@ def apply_cq(self, operations, **kwargs):
             invert_param = False
 
             for o in operations:
-                if o.base_name in skipped_ops:
+                if o.name in skipped_ops:
```

Comment:
```suggestion
                if str(o.name) in skipped_ops:
```

To handle the tensor edge-case.